.circleci/config.yml

```
@@ -196,26 +217,12 @@ jobs:
-              fi
-            done
-            
-            # Ensure we have at least one requirements file to scan
-            if [ -z "$REQUIREMENTS_ARGS" ]; then
-              echo "❌ No requirements files found to scan. Failing CI."
-              exit 1
-            fi
+            # Scan installed packages (which were installed with constraints applied)
+            # This ensures Safety checks against the exact constrained versions
+            echo "🔍 Scanning installed packages (constrained versions)..."
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)